assets/queries/k8s/volume_mount_with_OS_directory_write_permissions/metadata.json

```diff
@@ -0,0 +1,8 @@
+{
+  "id": "volume_mount_with_OS_directory_write_permissions",
+  "queryName": "volume mount with OS directory write permissions",
```

Comment:
Use capital letters in Query Name: "**V**olume **M**ount with OS **D**irectory **W**rite **P**ermissions

Reply:
This has been resolved!!